src/java/org/apache/cassandra/db/compaction/OperationType.java

```diff
@@ -50,15 +50,34 @@
     VIEW_BUILD("View build"),
     INDEX_SUMMARY("Index summary redistribution"),
     RELOCATE("Relocate sstables to correct disk"),
-    GARBAGE_COLLECT("Remove deleted data");
+    GARBAGE_COLLECT("Remove deleted data"),
+    RESTORE("Restore"),
```

Comment:
We should indicate somehow that these are required by CNDB or they may get removed by someone because they are not used anywhere.